logstash-core/src/main/java/org/logstash/Event.java

```diff
@@ -56,7 +57,7 @@ public Event()
      * @param data Map that is assumed to have either {@link String} or {@link RubyString}
      * keys and may contain Java and Ruby objects.
      */
-    public Event(Map data) {
+    public Event(final Map<String, Object> data) {
```

Comment:
The java doc claims either String or RubyString.  Does this break that contract ?

Reply:
@jakelandis urgh you're right ... we could still have `RubyString` here, but type erasure prevents this from throwing obviously :( 
=> gotta change this to `? extends Serializable`